setup/ci/index.md

```diff
@@ -24,5 +24,6 @@ These are the CI systems currently supported by Spinnaker:
 
 * [Jenkins](/setup/ci/jenkins/)
 * [Travis CI](/setup/ci/travis/)
+* [Wercker](/setup/ci/wercker/)
```

Comment:
Text above "Spinnaker supports two CI systems" should be changed to "three" (or just "several").

Reply:
done

Reply:
done